paasta_tools/utils.py

```diff
@@ -1214,6 +1297,28 @@ def log(
         formatted_line = format_log_line(level, cluster, service, instance, component, line)
         self.clog.log_line(log_name, formatted_line)
 
+    def log_audit(
+        self,
+        user: str,
+        host: str,
+        action: str,
+        action_details: dict = None,
+        service: str = None,
+        cluster: str = ANY_CLUSTER,
+        instance: str = ANY_INSTANCE,
+    ) -> None:
+        log_name = AUDIT_LOG_STREAM
+        formatted_line = format_audit_log_line(
+            user=user,
+            host=host,
+            action=action,
+            action_details=action_details,
+            service=service,
+            cluster=cluster,
+            instance=instance,
+        )
+        self.clog.log_line(log_name, formatted_line)
```

Comment:
How do you feel about _also_ logging to the service's log? I think a common use case will be to look at these audit logs for a particular service, e.g. with `paasta logs`.

Reply:
decided against this for now, requires some editorial decisions to the other log functionality that i'd just as soon skip for the moment